client/signup/steps/import-url/index.jsx

```diff
@@ -2,165 +2,279 @@
 /**
  * External dependencies
  */
-import React, { Component } from 'react';
+import React, { Component, Fragment } from 'react';
 import { localize } from 'i18n-calypso';
 import { connect } from 'react-redux';
-import { flow, get, indexOf, inRange } from 'lodash';
-import { isWebUri } from 'valid-url';
-import { parse as parseURL } from 'url';
+import { flow, isEqual } from 'lodash';
+import page from 'page';
 
 /**
  * Internal dependencies
  */
+import Button from 'components/button';
+import Card from 'components/card';
+import ExampleDomainBrowser from 'components/domains/example-domain-browser';
+import ExternalLink from 'components/external-link';
+import Notice from 'components/notice';
 import StepWrapper from 'signup/step-wrapper';
 import SignupActions from 'lib/signup/actions';
-import FormTextInputWithAction from 'components/forms/form-text-input-with-action';
-import { setNuxUrlInputValue, setValidationMessage } from 'state/importer-nux/actions';
+import FormTextInput from 'components/forms/form-text-input';
+import FormButton from 'components/forms/form-button';
 import FormInputValidation from 'components/forms/form-input-validation';
 import FormSettingExplanation from 'components/forms/form-setting-explanation';
+import { fetchIsSiteImportable, setNuxUrlInputValue } from 'state/importer-nux/actions';
 import {
+	getNuxUrlError,
 	getNuxUrlInputValue,
 	getSiteDetails,
-	getUrlInputValidationMessage,
 	isUrlInputDisabled,
 } from 'state/importer-nux/temp-selectors';
+import { validateImportUrl } from '../../../my-sites/importer/site-importer/url-validation';
```

Comment:
It's probably better to do:
`import { validateImportUrl } from 'my-sites/importer/site-importer/url-validation';`

...instead of so many `../`s

Reply:
Decided to move this to `lib/importers` and updated path.